yt_dlp/extractor/sportdeutschland.py

```diff
@@ -1,95 +1,110 @@
+            'channel_url': 'https://sportdeutschland.tv/blauweissbuchholztanzsport',
+            'channel_id': '93ec33c9-48be-43b6-b404-e016b64fdfa3',
+            'display_id': '9839a5c7-0dbb-48a8-ab63-3b408adc7b54',
+            'duration': 32447,
+            'upload_date': '20230114',
+            'timestamp': 1673730018.0,
+        }
     }, {
-        'url': 'https://sportdeutschland.tv/badminton/re-live-deutsche-meisterschaften-2020-halbfinals?playlistId=0',
+        'url': 'https://sportdeutschland.tv/deutscherbadmintonverband/bwf-tour-1-runde-feld-1-yonex-gainward-german-open-2022-0',
         'info_dict': {
-            'id': 'c6e2fdd01f63013854c47054d2ab776f',
-            'title': 'Re-live: Deutsche Meisterschaften 2020 - Halbfinals',
-            'description': 'md5:5263ff4c31c04bb780c9f91130b48530',
-            'duration': 31397,
-        },
-        'playlist_count': 2,
-    }, {
-        'url': 'https://sportdeutschland.tv/freeride-world-tour-2021-fieberbrunn-oesterreich',
-        'only_matching': True,
+            'id': '95b97d9a-04f6-4880-9039-182985c33943',
+            'ext': 'mp4',
+            'title': 'BWF Tour: 1. Runde Feld 1 - YONEX GAINWARD German Open 2022',
+            'description': 'md5:2afb5996ceb9ac0b2ac81f563d3a883e',
+            'live_status': 'was_live',
+            'channel': 'Deutscher Badminton Verband',
+            'channel_url': 'https://sportdeutschland.tv/deutscherbadmintonverband',
+            'channel_id': '93ca5866-2551-49fc-8424-6db35af58920',
+            'display_id': '95c80c52-6b9a-4ae9-9197-984145adfced',
+            'duration': 41097,
+            'upload_date': '20220309',
+            'timestamp': 1646860727.0,
+        }
     }]
 
     def _real_extract(self, url):
         display_id = self._match_id(url)
-        data = self._download_json(
-            'https://backend.sportdeutschland.tv/api/permalinks/' + display_id,
+        meta = self._download_json(
+            'https://api.sportdeutschland.tv/api/stateless/frontend/assets/' + display_id,
             display_id, query={'access_token': 'true'})
-        asset = data['asset']
-        title = (asset.get('title') or asset['label']).strip()
-        asset_id = asset.get('id') or asset.get('uuid')
+
+        asset_id = meta.get('id') or meta.get('uuid')
+        profile = meta.get('profile')
+
         info = {
             'id': asset_id,
-            'title': title,
-            'description': clean_html(asset.get('body') or asset.get('description')) or asset.get('teaser'),
-            'duration': int_or_none(asset.get('seconds')),
+            'title': (meta.get('title') or meta.get('name')).strip(),
+            'description': meta.get('description'),
+            'channel': profile.get('name'),
+            'channel_id': profile.get('id'),
+            'channel_url': 'https://sportdeutschland.tv/' + profile.get('slug'),
+            'is_live': meta.get('currently_live'),
+            'was_live': meta.get('was_live')
```

Comment:
```suggestion
        asset_id = traverse_obj(meta, 'id', 'uuid')
        info = {
            'id': asset_id,
            'channel_url': format_field(meta, ('profile', 'name'), 'https://sportdeutschland.tv/%s'),
            **traverse_obj(meta, {
                'title': (('title', 'name'), {strip_or_none}),
                'description': 'description',
                'channel': ('profile', 'name'),
                'channel_id': ('profile', 'id'),
                'is_live': 'currently_live',
                'was_live': 'was_live'
            }, get_all=False),
```

Reply:
Correct me if I'm wrong but is `traverse_obj` equal to an or operator? I thought it was equivalent to chaining `.get`


So 
```
profile = meta.get('profile')
[...]
'channel_id': profile.get('id'),
```

would become

```
'channel_id': traverse_obj(meta, ('profile', 'id'))
```

Reply:
That is one of it's uses. It does a lot of stuff to make safely traversing unstructured data easy https://github.com/yt-dlp/yt-dlp/blob/2e269bd998c61efaf7500907d114a56e5e83e65e/yt_dlp/utils.py#L5413-L5474

Reply:
ok, thanks for the info!